README.md

```diff
@@ -168,305 +116,247 @@ Typical Java threads
 
-Sometimes, for our own convenience, or for dealing with formats provided by up-stream systems,
-the data arrives packaged in various formats. These containers can have multiple items that
-should not be logically related in a family tree. The Emissary Parser framework deals with this
-situation.
+The Emissary Parser framework is designed to extract information from any data containers/wrappers on the payload that may contain information useful for identification or processing.
 
-Parsers need to make a quick identification of the format of the container. This is called
-out by the emissary.parser.ParserFactory and uses the emissary.parser.DataIdentifier as the
+Parsers need to make a quick identification of the container format (i.e. Apache NiFi). This is called
+out by the emissary.parser.ParserFactory and uses the [emissary.parser.DataIdentifier](src/main/java/emissary/parser/DataIdentifier.java) as the
 engine to perform the identification. The engine is configured by name and can be replaced
 by anything that meets the required interface. The name of the format identified should
 have parsers that meet the SessionParser interface configured into the factory.
 
 Once the container is identified, items are parsed out of it, built into IBaseDataObject instances
 by the SessionParser appropriate for that data type, and handed over to MobileAgent instances
-for processing as they become available from the pool. It is important to have in mind that the
+for processing as they become available from the pool. 
+
+It is important to have in mind that the
 Parser framework is operating on the single-threaded side of the system and handing items over to the
-multi-threaded (MobileAgent) side of the system. This implies that the parsers are fast enough
+multi-threaded (MobileAgent) side of the system. This necessitates that the parsers are fast enough
 to keep the number of configured threads busy.
 
 ## Development
 
 ### Requirements
 
-#### Linux or MacOSX operating system
-
-Currently, developing and running natively on Windows is not
-supported.  Use a virtualized Linux OS if you are using Windows.  
-
-In terms of Linux distributions, the two most widely used on the Emissary
-codebase are CentOS 6.X and Ubuntu 12.X.
-
 #### Java 1.8
 
-Java 1.6 and Java 1.7 are no longer supported in this codebase only 
-[Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html).  
-OpenJDK should work, although most testing is done with Oracle's JDK.  Be sure to use a recent version of
+Emissary requires [Java 1.8](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html).  
+Emissary now supports both OpenJDK and Oracle JDK.  Be sure to use a recent version of
 1.8 as there are bugs in earlier versions.
 
 #### Apache Maven 3.2
 
 Download and install [Apache Maven 3.2](http://maven.apache.org).
-Some maven commands are shown below.  A great place to start with Maven is the
```

Comment:
Condensed this info (see below). We aren't maven, so we can just point the user to good resources and save space in our README